Documentation/INTERNALS.md

```diff
@@ -165,18 +192,16 @@ cases:
  * If there are any fields containing a message or group type
  * If there are more than 16 total fields
 
+This logic will doubtless change in the future:
```

Comment:
Can't tell, did you mean for this to be a heading of a bullet list or something like that?